tests/filecheck/mlir-conversion/with-mlir/dialects/emitc/emitc_ops.mlir

```diff
@@ -0,0 +1,17 @@
+// RUN: xdsl-opt %s | xdsl-opt | mlir-opt --allow-unregistered-dialect | filecheck %s
```

Comment:
How come the ops below are printed in generic format in MLIR if there's a custom format?

Reply:
should be fixed with [c5606ae](https://github.com/xdslproject/xdsl/pull/4781/commits/c5606ae2417cf04a444f1220877358e3dfe568a4)

Reply:
Nice, let's replace this line with the following to make sure we're testing both generic and custom formats?
```suggestion
// RUN: xdsl-opt %s | mlir-opt | xdsl-opt | filecheck %s
// RUN: xdsl-opt %s --print-op-generic | mlir-opt --mlir-print-op-generic | xdsl-opt | filecheck %s
```